crates/egui-wgpu/src/renderer.rs

```diff
@@ -690,7 +690,7 @@ impl Renderer {
     ///
     /// This enables the application to reference the texture inside an image ui element.
     /// This effectively enables off-screen rendering inside the egui UI. Texture must have
-    /// the texture format [`wgpu::TextureFormat::Rgba8UnormSrgb`].
+    /// the texture format [`wgpu::TextureFormat::Rgba8Unorm`].
```

Comment:
huh I guess there's a future work item here that iff `Rgba8UnormSrgb` is passed in, we create an `Rgba8Unorm` view either way (telling the user that we do that)
If I remember correctly WebGPU standard now guarantees this working on the minspec.

Reply:
Can you even know what format the given texture has?

Reply:
https://github.com/gfx-rs/wgpu/pull/7895